### PR TITLE
docs: Provide a new documentation template

### PR DESCRIPTION
**Pull request by Mcalewaert**
Relates to https://github.com/arcus-azure/arcus/issues/113

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,8 +1,92 @@
...
+## Prerequisites
+This documentation assumes that you run all scripts within the docs folder.
 
-Running Jekyll locally to test your changes is super easy, learn more about it [here](https://jekyllrb.com/docs/#instructions).
\ No newline at end of file
+```console
+cd docs
+```
+
+
```
Comment (tomkerkhove) [inline docs/README.md:12]:
```suggestion

```

File: docs/README.md [hunk 2]
```diff
@@ -1,8 +1,92 @@
...
+When creating a new documentation website update these values within the `docusaurus.config.js` file:
+
+```typescript
+{
+  title,
+  url,
+  projectName,
+  themeConfig: {
+    navbar: {
+      title: 'Background Jobs',
```
Comment (tomkerkhove) [inline docs/README.md:59]: Copy paste

File: docs/README.md [hunk 3]
```diff
@@ -1,8 +1,92 @@
...
+        items: [
+        {
+          href: 'https://github.com/arcus-azure/arcus.backgroundjobs',
```
Comment (tomkerkhove) [inline docs/README.md:62]: Copy paste

File: docs/README.md [hunk 4]
```diff
@@ -1,8 +1,92 @@
...
+          label: 'GitHub',
+        }
+      ]
+    },
+    footer: {
+      copyright:  `Copyright © ${new Date().getFullYear()} Arcus Background Jobs maintained by arcus-azure`,
```
Comment (tomkerkhove) [inline docs/README.md:68]: Copy paste

File: docs/README.md [hunk 5]
```diff
@@ -1,8 +1,92 @@
...
+    }
+  },
+    presets: [
+    [
+      '@docusaurus/preset-classic',
+      {
+        docs: {
+          editUrl:
+            'https://github.com/arcus-azure/arcus.backgroundjobs/edit/master',
```
Comment (tomkerkhove) [inline docs/README.md:77]: Copy paste

File: netlify.toml [hunk 6]
```diff
@@ -1,36 +1,33 @@
 [Settings]
-# Added automatically by the Netlify CLI. It has no effect during normal 
-# Git-backed deploys.
-ID = "14115dc9-bace-418d-8120-2a5b4d9a43ce"
```
Comment (tomkerkhove) [inline netlify.toml:4]: On purpose?

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

File: netlify.toml (hunk 6 above)
Comment (Mcalewaert) [reply netlify.toml:4]: Kinda, doesn't this need a new id and site on netlify first?

Comment (tomkerkhove) [reply netlify.toml:4]: No, I will just rename the current staging one - No worries!

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**